Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/Profiles/DefaultSceneUnderstandingObserverProfile.asset

```diff
@@ -12,19 +12,20 @@ MonoBehaviour:
   m_Script: {fileID: 11500000, guid: 1a66f7659243ab748891f032ff4477e2, type: 3}
   m_Name: DefaultSceneUnderstandingObserverProfile
   m_EditorClassIdentifier: 
-  isCustomProfile: 1
-  startupBehavior: 1
+  isCustomProfile: 0
+  startupBehavior: 0
```

Comment:
Just double checking the change to startup behavior is intended.

Reply:
this was intended, yes :)